dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -147,26 +148,32 @@ class SentryStackTraceFactory {
     final scheme = frame.uri.scheme;
 
     if (scheme.isEmpty) {
-      return true;
+      // Early bail out.
+      return _options.isStackFrameInAppDefault;
     }
+    // The following code depends on the scheme being set.
 
-    for (final include in _options.inAppIncludes) {
-      if (frame.package != null && frame.package == include) {
+    final package = frame.package;
+    if (package != null) {
+      if (_options.inAppIncludes.contains(package)) {
```

Comment:
Should this be `startsWith` instead?

Reply:
Could you elaborate on that? I think I don't get what you'r hinting at.

Though I'm pretty sure it shouldn't be a `startsWith`.
`inAppIncludes` is a list of Strings (package names) which should be considered as in app frames.
So I think the name should match exactly. If we're using `startsWith` and use `flutter` as an include
the `flutter_svg` package would also be considered in app.

Reply:
on Flutter and Dart it makes sense to match exactly because we match only the package which isn't hierarquical.
on Java/Android is the package, which is like `io.sentry` so  `startsWith` makes sense because I may have `io.sentry.a` and `io.sentry.b` but I don't want to maintain a list of my endless packages, so I just add the root of my package.

Reply:
RIght, I didn't think this through. Java and .NET is `startsWith`. But here as @marandaneto indicated, I believe it should be an exact match as opposed to `contains`

Reply:
the `contains` here does an exact match, the `contains` is in the list instance and not in the string, so it's correct already